src/components/App.jsx

```diff
@@ -9,6 +9,7 @@ import {
   appointmentRoutes,
   manageAppointmentRoutes,
 } from 'components/appointments';
+import formRoutes from 'components/formRoutes';
```

Comment:
once we're around to this refactoring, I want to move the route definition files too, since this is not a component :)